### PR TITLE
Fix HTML markup errors

### PR DESCRIPTION
**Pull request by abitrolly**
Use https://validator.w3.org/nu/?doc=https%3A%2F%2Ffwupd.org%2F for more.

I just edited files from GitHub editor. No time to roll out development environment for testing.

### CODE REVIEW

File: app/templates/default.html [hunk 1]
```diff
@@ -155,17 +157,17 @@ <h5 class="modal-title" id="loginModalLabel">Do you really need to log in to the
...
           <a class="btn btn-secondary btn-block" href="/lvfs/login/auth-azure">
-            <img class="mr-1" src="/img/microsoft_logo_btn.png"/>
+            <img class="mr-1" src="/img/microsoft_logo_btn.png" alt=""/>
             <b>Log in with Azure AD</b>
           </a>
           <hr/>
           <p class="text-secondary">If you have a local account please log in below:</p>
           <div class="form-group">
-            <label for="group_id">Email Address:</label>
+            <label>Email Address:</label>
```
Comment (hughsie) [inline app/templates/default.html]: I thought the `for=` has to point to the `<input>`?

**Comment by abitrolly** [comment]
I was too lazy. Should be better now. :) Test are failing because of broken yum mirrors. No idea how to fix that - I don't use `yum`.

**Comment by hughsie** [comment]
Squashed and merged, thanks!

**Comment by abitrolly** [comment]
124bc2671b12. Everything is good now - https://validator.w3.org/nu/?doc=https%3A%2F%2Ffwupd.org%2F - but it is only one page. :)

**Pull request closed without merging**